server/analytic.go

```diff
@@ -1,64 +0,0 @@
-package main
-
-import (
-	"sync"
-	"time"
-)
-
-type Analytic struct {
-	lock          sync.RWMutex
-	Start         time.Time
-	End           time.Time
-	Channels      map[string]int64
-	ChannelsReply map[string]int64
-	Users         map[string]int64
-	UsersReply    map[string]int64
-	FilesNb       int64
```

Comment:
Should this be `FilesMb`?

Reply:
No it's the number of files downloaded, the next property of this struct `FilesSize` represent the weight of all files.

Reply:
For what does `Nb` stand for?

Reply:
As the future godoc say ( :D ) : `FilesNb store number of files uploaded`

![selection_197](https://user-images.githubusercontent.com/1492516/53494144-5b723a80-3a9d-11e9-8ac0-3f922a592de5.png)

![selection_198](https://user-images.githubusercontent.com/1492516/53494350-d63b5580-3a9d-11e9-9e81-5a67d510c121.png)

I'm sorry my english is not fluent :sweat_smile: maybe better name can be found